server/src/main/java/io/pravega/schemaregistry/server/rest/ServiceConfig.java

```diff
@@ -28,25 +28,30 @@
     @ToString.Exclude
     private final String tlsCertFilePath;
     @ToString.Exclude
-    private final String serverKeyStoreFilePath;
+    private final String tlsKeyStoreFilePath;
     @ToString.Exclude
     private final String tlsKeyStorePasswordFilePath;
     private final boolean authEnabled;
     @ToString.Exclude
     private final String userPasswordFilePath;
 
     private ServiceConfig(String host, int port, boolean tlsEnabled, String tlsCertFilePath, 
-                          String serverKeyStoreFilePath, String tlsKeyStorePasswordFilePath, boolean authEnabled, String userPasswordFilePath) {
+                          String tlsKeyStoreFilePath, String tlsKeyStorePasswordFilePath, boolean authEnabled, String userPasswordFilePath) {
         Exceptions.checkNotNullOrEmpty(host, "host");
         Exceptions.checkArgument(port > 0, "port", "Should be positive integer");
-        Exceptions.checkArgument(!tlsEnabled || (!Strings.isNullOrEmpty(tlsCertFilePath) &&
-                        !Strings.isNullOrEmpty(serverKeyStoreFilePath)), "keyFilePath", 
-                "If tls is enabled then key file path and key file password path should be non empty");
+        Exceptions.checkArgument(!tlsEnabled || !Strings.isNullOrEmpty(tlsCertFilePath), "keyCertPath", 
+                "If tls is enabled then cert file path should be non empty");
+        Exceptions.checkArgument(!tlsEnabled || !Strings.isNullOrEmpty(tlsCertFilePath), "keyCertPath", 
```

Comment:
It could be simplified further by doing something like this: 

```
if (tlsEnabled) {
     Exceptions.checkNotNullOrEmpty(tlsCertFilePath)....
}
```